.github/CODEOWNERS

```diff
@@ -2,3 +2,6 @@
 # .github/workflows directory (and the merge logic) using CODEOWNERS.
 /.github/workflows/ @klizhentas @russjones @r0mant @zmb3 @fheinecke @camscale @tcsc @rosstimothy
 /build.assets/tooling/cmd/difftest/ @klizhentas @russjones @r0mant @zmb3
+
+# owners for dependency updates in /web packages
+**/package.json @avatus @gzdunek @ravicious
```

Comment:
I'm not sure if this will match `package.json` in the root dir? I think `package.json` would watch any package.json file in any dir.